### DISCUSSION TITLE
API Key Improvements

### DISCUSSION THREAD

**Discussion by stgarf-sx** [DependencyTrack/dependency-track#2720] [Ideas]
Would love to see the [API key management improvements mentioned here](https://github.com/DependencyTrack/dependency-track/issues/2552) (plus [some of the other ones suggested as issues](https://github.com/stevespringett/Alpine/issues/362) related to API keys in the [Alpine repo](https://github.com/stevespringett/Alpine)) implemented. Can you provide any information regarding if that'd be on the roadmap or not?

And on the topic of roadmaps -- is there a rough one to follow to see upcoming/planned features/enhancements?

**Comment by syalioune** [comment]
Hello @stgarf-sx, you can take a look at the [milestones](https://github.com/DependencyTrack/dependency-track/milestones) page for a rough roadmap.
For anything not already planned in a milestone, community help is welcome 😃
